src/power_grid_model/data_types.py

```diff
@@ -25,255 +64,223 @@
     - "u_rated"
 """
 
-SparseDataComponentType: TypeAlias = str
-"""
-A string representing the component type of sparse data structures.
-
-Must be either "data" or "indptr".
-"""
-
-SingleArray: TypeAlias = np.ndarray
```

Comment:
here and below: please double-check that all of these symbols and their docstrings are still present